org.eclipse.swtchart.cbi/pom.xml

```diff
@@ -178,6 +183,8 @@
         <version>${tycho.version}</version>
         <configuration>
           <useUIHarness>true</useUIHarness>
+          <argLine>-XstartOnFirstThread</argLine>
```

Comment:
> Unrecognized option: -XstartOnFirstThread

Reply:
Okay, I was hoping that the option is just ignored on other platforms than macOS.
I do not really understand why Tycho surefire is not adding this as needed.

Reply:
For now I will solve it locally by running the tests with

```
mvn -f org.eclipse.swtchart.cbi/pom.xml -Dtycho.testArgLine=-XstartOnFirstThread clean verify
```